crypto/src/mls/external_commit.rs

```diff
@@ -262,6 +269,22 @@ impl MlsConversation {
             }
         }
 
+        if let Some(pki_env) = pki_env {
+            let credentials: Vec<_> = commit
+                .add_proposals()
+                .filter_map(|add_proposal| {
+                    let credential = add_proposal.add_proposal().key_package().leaf_node().credential();
+
+                    matches!(credential.credential_type(), CredentialType::X509).then(|| credential.clone())
+                })
+                .collect();
+            let state = compute_state(credentials.iter(), Some(pki_env), MlsCredentialType::X509);
+            if state != E2eiConversationState::Verified {
+                // FIXME: Uncomment when PKI env can be seeded - the computation is still done to assess performance and impact of the validations
+                // return Err(CryptoError::InvalidCertificateChain);
```

Comment:
I guess this can be uncommented now ?

Reply:
> I guess this can be uncommented now ?

This will make *all* tests fail. We need to uncomment it for sure only when we have a test environment for this.